cpp/src/backends/protocol/otf_message.hh

```diff
@@ -28,7 +28,9 @@ namespace torchserve {
 
   class OTFMessage {
     public:
-    static byte_buffer CreateLoadModelResponse(StatusCode code, const std::string& message);
+    static bool SendAll(Socket conn, char *data, size_t length);
+    static void CreateLoadModelResponse(std::unique_ptr<torchserve::LoadModelResponse> response, char* data);
```

Comment:
I think `EncodeLoadModelResponse` makes more sense since we are just encoding an existing response.

Reply:
Refactored